.github/workflows/ci.yml

```diff
@@ -1695,8 +1696,7 @@ jobs:
       DEBUG_RESOURCES: ${{needs.build-info.outputs.debug-resources}}
     if: >
       ( needs.build-info.outputs.run-kubernetes-tests == 'true' ||
-      needs.build-info.outputs.needs-helm-tests == 'true' ) &&
-      needs.build-info.outputs.default-branch == 'main'
+      needs.build-info.outputs.needs-helm-tests == 'true')
```

Comment:
BTW. This is needed because I realized we should run k8s tests also on `v2_*-test` branches even if we do not release Helm Chart there, it is great to verify that our "about to be released" version of airflow works end-to-end in K8S.

Reply:
I could separate that one out, but It's a small thing that does not even affect `main` branch - just future v2* branches